library/src/scala/tasty/Reflection.scala

```diff
@@ -1353,8 +1353,13 @@ trait Reflection extends reflect.Types { reflectSelf: CompilerInterface =>
     def of[T <: AnyKind](using qtype: scala.quoted.Type[T], ctx: Context): Type =
       qtype.asInstanceOf[scala.internal.quoted.Type[TypeTree]].typeTree.tpe
 
-    def apply(clazz: Class[_])(using ctx: Context): Type =
-      reflectSelf.Type_apply(clazz)
+    /** Returns the type of the runtime class. This type is the erased representation
+     *  that of the type that is used by arrays.
+     *
+     *  See `Type.of` to get the true type.
+     */
+    def ofErasedClass(clazz: Class[_])(using ctx: Context): Type =
```

Comment:
Is this actually used by anything else than `scala.quoted.ClassLiftable`? If not, could we make it `private[scala]` unless and until we discover an imperative use case outside?

Reply:
Good point.

Reply:
I don't see any uses outside lifting arrays